### PR TITLE
Addisie md dl002

### PR DESCRIPTION
**Pull request by thea-m**
@Addisie has a technical problem logging in into GitHub due to 2fA while in Ethiopia. She has done the description and will read and act on all comments and corrections. (I am just the messenger)

### CODE REVIEW

**Comment by eu-genia** [comment]
Thank you @Addisie !
Maybe @Mersha-Alehegne will be able to provide some of the missing metadata.
I have checked and it is the file that had been previously shared  (as PDF) with the HLCEES, at that time the folder name was "Mamher Demisse"
Now thaI assume, as discussed with @Mersha-Alehegne , that the abbreviation MD stands actually for Mamher Deggu https://hviewer.bl.uk/IamsHViewer/Default.aspx?mdark=ark:/81055/vdc_100023431294.0x000001&_ga=2.108858306.1304144530.1586083190-448041624.1571936776
, a collection photographed for EAP. Yet unfortunately this particular MS does not seem to be among the ones listed at the EAP (or submitted), as far as I could check, and it is not present in the metadata spreadsheet that I received from @Mersha-Alehegne  either.

File: new/MDDL002.xml [hunk 1]
```diff
@@ -0,0 +1,445 @@
...
+                            <supportDesc>
+                                <support>
+                                    <material key="parchment"/>
+                                </support>
+                                <extent>
+                                    <measure unit="leaf" quantity="186">186</measure>
+                                    <measure unit="page" type="blank">4</measure>
+                                    <locus from="1r" to="1v"/>
+                                    <locus target="#165v #186v"/>
+                                   <note>Measurements of the manuscript's dimensions were not provided in the manuscript's metadata and images.</note>
```
Comment (eu-genia) [inline new/MDDL002.xml:201]: maybe dimensions can be estimated using the paper ruler photographed together with the MS

Comment (eu-genia) [reply new/MDDL002.xml:201]: I have now used https://imagemeasurement.online/ applying the scale provided and can give the 

outer measurements
height: 326 mm
width: 286 mm

text area (f. 5r)
height: 205 mm
width: 215 mm

File: new/MDDL002.xml [hunk 2]
```diff
@@ -0,0 +1,445 @@
...
+                                </item>
+                                <item xml:id="e5">
+                                    <desc><locus target="#166r"/> The title  <foreign xml:lang="gez">ተረፈ፡ ቄርሎስ፡</foreign> written in the 
+                                        upper margin.
+                                    </desc>
+                                </item>
+                                <item xml:id="e6">
+                                    <desc><locus target="#4rc"/> scribbles in the margin.
+                                    </desc>
+                                </item>
```
Comment (eu-genia) [inline new/MDDL002.xml:391]: maybe the title on the mahdar (አፈወርቅ፡) should also be listed as an e7 or in the binding description

File: new/MDDL002.xml [hunk 3]
```diff
@@ -0,0 +1,445 @@
...
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="MDDL002" xml:lang="en" type="mss">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title>Tǝrgʷāme malʾǝkt za-bǝṣuʿ wa-kǝbur Ṗāwlos za-ṣaḥafa la-ʿƎbrāwǝyān, Qerǝllos</title>
+                <editor role="generalEditor" key="AB"/>
+                  <editor key="AY"/>
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <editionStmt><p>Digitized and catalogued within the framework of the project Documenting an
```
Comment (eu-genia) [inline new/MDDL002.xml:13]: this probably has to be changed, as the MS was digitized much earlier, within the framework of EAP336 project - but it was not submitted among the EAP mss, so I am not sure; possibly we should simply write that it was digitized by @Mersha-Alehegne @MershaMengistie in 2010-2011 and catalogued within the framework of the ʾAbǝnnat Tǝmhǝrt project

**Review by eu-genia — CHANGES_REQUESTED** [review]

File: new/MDDL002.xml [hunk 4]
```diff
@@ -198,7 +198,11 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                                     <measure unit="page" type="blank">4</measure>
                                     <locus from="1r" to="1v"/>
                                     <locus target="#165v #186v"/>
-                                   <note>Measurements of the manuscript's dimensions were not provided in the manuscript's metadata and images.</note>
+                                    <note>Measurements of the manuscript's dimensions were not provided in the manuscript's metadata and images, they were measured digitally.</note>
+                                    <dimensions type="outer" unit="mm">
+                                        <height>326 </height>
```
Comment (eu-genia) [inline new/MDDL002.xml:203]: please remove the blank space after 326

**Comment by thea-m** [comment]
Thank you @eu-genia for the comments and the helpful measuring tool, I did not know about it

**Review by eu-genia — APPROVED** [review]

**Comment by MershaMengistie** [comment]
I have one suggesion Zhenia: until I come and clarify the problems with the EAP 336 mss, Addisie can work with the recently documented mss as they are not complicated like the ones on the table. Of course the older ones should also be finalized but later.  Just a suggestion.

**Pull request merged**